docs/DeveloperGuide.md

```diff
@@ -856,3 +856,10 @@ testers are expected to do more *exploratory* testing.
        e.g. add a non-alphanumeric character to one of the student's name.<br>
        Expected: UGTeach should discard all data in the file and start with an empty `ugteach.json` file.
 
+--------------------------------------------------------------------------------------------------------------------
+
+## **Appendix: Planned Enhancements**
+
+**_Team size: 5_**
+
+1. **Improve duplicate detection to meet real-world use cases**: The current version only considers students with **both** same name and same phone number as duplicates, since we take the 2 fields as main criteria to differentiate the students while not restricting users to only adding students with different names. However, email address is also a unique identity. <br> In the future version, we plan to include email address as another criteria for detecting duplicates.
```

Comment:
> while not restricting users to only adding students with different names

Sorry I'm a bit confused, what does this means?

Reply:
Actually, i want to say that we consider the phone as duplicate criteria so that if there are 2 students with same name, but they are actually different, the user can still add them in (as they have diff phone num)

As for the old ab3, name is the only criteria, so cannot hv 2 ppl w same name. But the thing is that NAME in ab3 is not case sensitive, so users can still add in same name lowercase, for example.

Reply:
ohh, perhaps can rephrase `we take the 2 fields as main criteria to differentiate the students while not restricting users to only adding students with different names` as 
`we take the 2 fields as the main criteria to differentiate the students, since different students might have the exact same name`